pkg/systemd/quadlet/quadlet.go

```diff
@@ -2249,3 +2272,35 @@ func addVolumes(quadletUnitFile, serviceUnitFile *parser.UnitFile, groupName str
 
 	return nil
 }
+
+// Add hosts to the podman command
+// An IP can refer to one or multiple hosts following the format :
+// my-host-name;my-second-host-name;third-hostname:192.168.10.10
```

Comment:
What is the point of inventing a new syntax? If you want multiple hosts entries specify them several times.
I am not sure why this extra complexity is needed over the simple
```
addHosts := quadletUnitFile.LookupAll(groupName, KeyAddHost)
for _, addHost := range addHosts {
	podman.add("--add-host=" + addHost)
}
```

Reply:
We have an application which needs a lots of entry in the host files and multiple hostname refer to the same IP. 

With kube play (or also directly by editing the hosts file), we can set multiple hostname for an IP without duplicating that IP. I think it helps with the clarity of the file. 

However, I agree that it would be better if it wasn't a quadlet specific syntax. Beginning of next week I will remove this specificity from the PR.